Gemfile

```diff
@@ -12,6 +12,9 @@ gem "sprockets-rails"
 # Use postgres as the database
 gem "pg"
 
+# For Jiminy / Prosopite
+gem 'pg_query'
```

Comment:
I didn't realise this is a requirement for non-mysql environments. Would be a good idea to make it a dependency of Jiminy with optional loading. WDYT?

Reply:
Makes sense I think! 

> with optional loading

What does that look like, is it in the gemspec or is it like the `if defined? RSpec` sort of pattern?